python/TestHarness/TestHarness.py

```diff
@@ -468,7 +468,7 @@ def buildPBSBatch(self, output, tester):
       return ('QSUB NOT FOUND', '')
     else:
       # Get the PBS Job ID using qstat
-      results = re.findall(r'JOB_NAME: (\w+\d+) JOB_ID: (\d+) TEST_NAME: (\S+)', output, re.DOTALL)
+      results = re.findall(r'JOB_NAME: (\w+) JOB_ID:.* (\d+).*TEST_NAME: (\S+)', output, re.M)
```

Comment:
I think... I was _thinking_ I was being smart... I know I need the regex changes, but not the global regex modifier. I probably thought DOTALL did -not- include multilines, when the cluster_launcher is returning them:
```
cluster_launcher.py pbs_001.cluster
gold
JOB_NAME: simple_00_004 JOB_ID: 244367 TEST_NAME: kernels/simple_diffusion.test
```

In any case, I am cleaning up this junk and will have a stronger pull request shortly!